src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/ReportRepository.kt

```diff
@@ -194,6 +198,32 @@ class OrderlyReportRepository(val isReviewer: Boolean,
         }
     }
 
+    override fun setGlobalPinnedReports(reportNames: List<String>)
+    {
+        JooqContext().use { ctx ->
+            ctx.dsl.transaction { _ ->
+                ctx.dsl.deleteFrom(ORDERLYWEB_PINNED_REPORT_GLOBAL)
```

Comment:
I don't think this is using the transaction scope created - you have to create a new dsl object from the returned configuration, see: https://github.com/vimc/orderly-web/blob/master/src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/TagRepository.kt#L52 and documentation: https://www.jooq.org/doc/3.13/manual/sql-execution/transaction-management/)